frame/examples/basic/Cargo.toml

```diff
@@ -16,9 +16,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false }
 scale-info = { version = "2.5.0", default-features = false, features = ["derive"] }
 log = { version = "0.4.17", default-features = false }
-frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, path = "../../benchmarking" }
-frame-support = { version = "4.0.0-dev", default-features = false, path = "../../support" }
-frame-system = { version = "4.0.0-dev", default-features = false, path = "../../system" }
+frame = { version = "0.0.1-dev", default-features = false, path = "../../../frame/" }
```

Comment:
If we are coming this far, should we refactor the whole example to also not use sp-* as well? everything should be re-exported from frame. 

Or else, we can leave pallet-example-basic as is for now, and add benchmarking to pallet-example-frame-api.

Reply:
> Or else, we can leave pallet-example-basic as is for now, and add benchmarking to pallet-example-frame-api.

maybe it is better to leave pallet-example-basic as is for now, I used it only as a tests, if we move basic example to frame we shouldn't use the re-export `frame::deps` so much also, it needs to be cleaned

Reply:
Agreed, then let's do all experiments in `pallet-example-frame-api` for now.